.github/workflows/test.yml

```diff
@@ -1,40 +1,39 @@
-name: test
-on: [push]
+name: CI
+
+on: [push, pull_request]
+
 jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
There's a [`checkout@v2`](https://github.com/actions/checkout/blob/main/CHANGELOG.md) of this action you might want to "check out".